### PR TITLE
Documentation of the APis

### PR DESCRIPTION
**Pull request by Csasaka19**
Feat: Added a schema documentation path provided by django rest framework

### CODE REVIEW

File: daystar_orientation/daystar_orientation/urls.py [hunk 1]
```diff
@@ -9,4 +10,6 @@
     path('activities/', include('activities.urls')),
     path('events/', include('events.urls')),
     path('notifications/', include('notifications.urls')),
+    path('schema/', SpectacularAPIView.as_view(), name='schema'),
```
Comment (eiidoubleyuwes) [inline daystar_orientation/daystar_orientation/urls.py:13]: Looks good,Will test these new changes once code is live

**Review by eiidoubleyuwes — APPROVED** [review]
Looks good!

**Comment by Csasaka19** [comment]
I think they will be using it tomorrow.

**Comment by Csasaka19** [comment]
Were migrations for the account app done again ? It seems the campus field is not created for the user

**Comment by Csasaka19** [comment]
I also added a relationship for the parents and children so I think the migrations will rectify this.

**Pull request merged**